src/tests/multihost/admultidomain/readme.rst

```diff
@@ -0,0 +1,133 @@
+AD Provider Test Suite
+======================
+
+This directory contains test automation for SSSD AD Provider.
+
+
+Fixtures
+========
+
+
+session
+*******
+
+* setup_session: This fixtures does the following tasks:
+
+
+  * Install common required packages like
+  * Updated /etc/resolv.conf with Windows IP Address
+  * Clear sssd cache
+  * Configure system to use sssd authentication
+
+
+* teardown_session: This is not a fixtures but a teardown of ``setup_session``
+
+  * Restores resolv.conf
+  * Stop sssd service
+  * remove sssd.conf
+
+
+class
+*****
+
+* multihost: This fixture returns multihost object. Also using builtin request
+  fixture we pass ``class_setup`` and ``class_teardown``.  If the test suite defines
+  class_setup and class_teardown functions, multihost object will be available
+  to execute any remote functions.
+
+* clear_sssd_cache: Stops sssd service. Removes cache files from
+  ``/var/lib/sss/db`` and starts sssd service. Sleeps for 10 seconds.
+
+* enable_autofs_schema: Backup sssd.conf and Edit sssd.conf and specify
+  ``autofs_provider = ad`` and ``debug_level = 9``
+
+* enable_ad_sudoschema: Enable AD Sudo Schema
+
+* create_ad_sudousers: Create users in Windows Active Directory with username
+  from ``sudo_idmuser1`` to ``sudo_idmuser10``.
+
+* sudorules: Create AD sudo rules ``less_user_rule1`` to ``less_user_rule10``::
```

Comment:
Do we need the actual ldap entry in the readme?

Reply:
Removed.